app/javascript/dashboard/i18n/locale/de/campaign.json

```diff
@@ -2,11 +2,14 @@
   "CAMPAIGN": {
     "HEADER": "Kampagnen",
     "SIDEBAR_TXT": "Proaktive Nachrichten erlauben dem Kunden, ausgehende Nachrichten an seine Kontakte zu senden, was weitere Unterhaltungen auslösen würde. Klicke auf <b>Kampagne hinzufügen</b> um eine neue Kampagne zu erstellen. Sie können auch eine bestehende Kampagne bearbeiten oder löschen, indem Sie auf die Schaltfläche Bearbeiten oder Löschen klicken.",
-    "HEADER_BTN_TXT": "Kampagne erstellen",
+    "HEADER_BTN_TXT": {
+      "ONE_OFF": "Create a one off campaign",
+      "ONGOING": "Create a ongoing campaign"
```

Comment:
@sojan-official I am aware that the PR may not be ready for the review yet.

But should it be `"Create an ongoing campaign"`?